Userland/Libraries/LibWeb/HTML/HTMLSelectElement.cpp

```diff
@@ -46,6 +46,11 @@ void HTMLSelectElement::visit_edges(Cell::Visitor& visitor)
     visitor.visit(m_options);
     visitor.visit(m_inner_text_element);
     visitor.visit(m_chevron_icon_element);
+
+    for (auto const& item : m_select_items) {
+        if (item.has<SelectItemOption>())
+            visitor.visit(item.get<SelectItemOption>().option_element);
```

Comment:
If the item contains `SelectItemOptionGroup`, you will also need to visit each `option_element` in the `Vector<SelectItemOption>` list.